### PR TITLE
promote memory qos to beta

### PR DESCRIPTION
**Pull request by pacoxu**
- [ ] merge after https://github.com/kubernetes/kubernetes/pull/118699

https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2570-memory-qos: https://github.com/kubernetes/enhancements/issues/2570

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (Shubham82) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:148]: @pacoxu, mention the release up to which `MemoryQoS` was in alpha.

 ```suggestion
| `MemoryQoS` | `false` | Alpha | 1.22 | 1.27 |
```

**Review by Shubham82 — CHANGES_REQUESTED** [review]

**Comment by Shubham82** [comment]
put on hold until [k8s/k8s #118699](https://github.com/kubernetes/kubernetes/pull/118699) will merge.

/hold

Please remove the hold label once the above PR will merge.

**Comment by Shubham82** [comment]
Thanks, @pacoxu 
/lgtm

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Comment by sftim** [comment]
Is this change still needed for v1.28?

**Comment by sftim** [comment]
Because https://github.com/kubernetes/kubernetes/pull/118699 is marked as postponed:
/approve cancel

Feel free to re-approve if context shows that's the right thing to do.

**Comment by pacoxu** [comment]
> Is this change still needed for v1.28?

No, we decided to postpone the promotion due to a throttling problem that will make application stuck and no successful response later( which is worse than immediately oom kill).

**Comment by pacoxu** [comment]
/close
I will open another one if we want to promote it later.

**Pull request closed without merging**
